Makefile

```diff
@@ -53,19 +55,24 @@ code/run/service_account: setup/service_account
 	$(MAKE) code/run
 
 .PHONY: code/compile
-code/compile:
+code/compile: code/gen
 	@GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o=$(COMPILE_TARGET) ./cmd/manager
 
+deploy/crds/integreatly.org_installations_crd.yaml: pkg/apis/integreatly/v1alpha1/installation_types.go
+	operator-sdk generate openapi
+
+pkg/apis/integreatly/v1alpha1/zz_generated.deepcopy.go:	pkg/apis/integreatly/v1alpha1/installation_types.go
```

Comment:
@HubertStefanski maybe we need to duplicate this one to have `zz_generated.openapi.go` as a target also, WDYT?

Reply:
Makes sense, as it's dependent on the same types.go file, Will do!